migrations/alembic.ini

```diff
@@ -0,0 +1,45 @@
+# A generic, single database configuration.
```

Comment:
when did we add alembic? what is its use case?

Reply:
I migrate our old user db to a new table without drop any user data, and I google people use alembic to do this work safely.